### PR TITLE
Prepare 0.75.2 release

### PR DESCRIPTION
**Pull request by dreamer**

### CODE REVIEW

**Comment by dreamer** [comment]
@kcgen Still waiting for feedback :)

**Comment by kcgen** [comment]
Darn, this didn't show up in my notifications (or maybe was buried!). Thanks for the ping; reviewing!

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: The color palette for monochrome machines, such as `machine = hercules` or `machine = cga_mono`, can now to be set to either `white`, `paperwhite`, `green`, or `amber` in the configuration file, as follows:

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: from **the** 0.75.x 
from **the** upcoming

Comment (kcgen) [inline content/v0.75.2.md]: Consistency suggestion:

... release from the **_stable_** 0.75.x `<something>`
... backported from the **_upcoming_** 0.76.x `<something>`

Where `<something>` is  _release_ or _branch_ or .. other name? I'm not set on specific one.. just that we use the same terminology to refer in both cases.

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: to bypass **the** operating

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: mouses -> mice

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md:74]: very nice!

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: have **a** few

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: _suggest keeping_

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: ... _- some existing and popular programs, such as Wine and GUI frontends, depend on the `bosbox` binary name._

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: (minor adjustment)
- You don't need to generate your own desktop entry, appstream metadata (metainfo.xml), **or** package description: please see the relevant files in the `contrib/linux/` directory.

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: - We provide desktop icons in scalable or raster formats including those optimized for small sizes. Find them in the `contrib/icons/` directory.

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: flags (it's important!), **as well as using** `-O3` ...

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: (drop the comma)
Our testing _indicates that_ ...

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: **Documentation Contributors:**
- nemo93
- MasterO2
- Patryk Obara
- Wengier
- kcgen

_(**internal comment:** placed in order of contributions.. nemo93 made 50 wiki contributions between 0.75.1, Master02 made 23, .. awesome!)_

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md:109]: ## 5. Contributions during the 0.75.2 period

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md:115]: Maybe "PR Contributors:" (to differentiate from Documentation)?

File: content/v0.75.2.md
Comment (kcgen) [inline content/v0.75.2.md]: The detailed per-commit changelog can be read using the following Git command:

**Review by kcgen — CHANGES_REQUESTED** [review]
config.toml contains `title = "dosbox-staging"`
Should that be updated DOSBox Staging too? (I'm not sure).

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: I think `machine` setting name is confusing to users (not everyone will know Hercules with graphics card, the setting suggests it is a type of computer), so how slightly different wording:

"When emulating monochrome displays (via `machine = hercules` or `machine = cga_mono`), the default colour palette can now be set to either `white`, `paperwhite`, `green`, or `amber` in the configuration file, as follows:

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: Thanks :)

File: content/v0.75.2.md
Comment (kcgen) [reply content/v0.75.2.md]: Even better; thanks for the refinement @dreamer.

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: Hmm, I was trying to be precise, as we don't have 0.76.x release branch yet. Maybe I'll just drop the mention of backport - it does not add anything new to the release notes.

Comment (kcgen) [reply content/v0.75.2.md]: > we don't have 0.76.x release branch yet

That's ok, because _upcoming_ implies that whatever you want to call it, that that's still to-come.

It's fine to say:

from the stable 0.75.x release.. back-ported from the upcoming 0.76.x release.

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: I thought "nor" is the preferred one in the "negative" context? 

I applied fix to the rest of the sentence.

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: Changed the sentence a little bit:
```
- We provide desktop icons in scalable or raster formats (optimized for small
  sizes).  Find them in the `contrib/icons/` directory.
```

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: @nemo93 accepted the invite, so I will place him in the Maintainers section instead :)

I will add @MasterO2 to the contributors - I don't want to split contributors into two lists, as both code and documentation contributions are equally valuable :)

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md:109]: I really don't want to do that, as it's yet another place where I need to update the version… I think it's implied.

File: content/v0.75.2.md
Comment (kcgen) [reply content/v0.75.2.md]: When you have a negative introduction, _"you don't need to ..."_ I think of that as a -1 multiplied by everything in the subsequent list, which is simply a, b, and c; or a, b, or c.

But when you start your list (or sentence) with "neither", then "nor" carries the negative through to the end.  _Neither option A nor B were suitable._ 

The "neither" can be implied:  _I prefer not to compile with Clang, nor do I like to compile with VisualStudio._ If we rephrased it slightly then neither becomes apparent: _I prefer to compile with neither Clang nor Visual Studio._

That said.. I try to avoid neither-nor, as the sentence can almost always be rephrased without it and there are fewer negatives or double-negatives to keep track of :-).

File: content/v0.75.2.md
Comment (kcgen) [reply content/v0.75.2.md:109]: Roger that!

File: content/v0.75.2.md
Comment (dreamer) [reply content/v0.75.2.md]: After a bit of consideration… I decided to drop this section altogether. It's a leftover from a time when we had a long list of commits in the release notes. Technical and interested users probably know how to use git log on their own; for non-technical users it's pointless distraction.

**Comment by dreamer** [comment]
OK, I think it's done; preparing to make the release (but feedback on the release notes is still appreciated) :).

**Review by kcgen — APPROVED** [review]
Thanks for preparing the release notes @dreamer!

**Pull request merged**
